src/views/editor/keypress/key.rs

```diff
@@ -104,8 +106,8 @@ impl KeyInput {
             "shift" => (Key::Named(Shift), PhysicalKey::Code(KeyCode::ShiftLeft)),
             "symbol" => (Key::Named(Symbol), PhysicalKey::Code(KeyCode::Fn)),
             "symbollock" => (Key::Named(SymbolLock), PhysicalKey::Code(KeyCode::Fn)),
-            "hyper" => (Key::Named(Hyper), PhysicalKey::Code(KeyCode::Hyper)),
-            "super" => (Key::Named(Super), PhysicalKey::Code(KeyCode::Meta)),
+            "hyper" => (Key::Named(Meta), PhysicalKey::Code(KeyCode::MetaLeft)), // FIXME!
+            "super" => (Key::Named(Meta), PhysicalKey::Code(KeyCode::MetaLeft)), // FIXME!
```

Comment:
```suggestion
            "hyper" => (Key::Named(Meta), PhysicalKey::Code(KeyCode::MetaLeft)),
            "super" => (Key::Named(Meta), PhysicalKey::Code(KeyCode::MetaLeft)),
```
These look fine. I don't think they need a fix